they changed the install for this but not the rpath added to any
executable, which still is the rootlibexecdir
--
diff --git a/src/shared/meson.build b/src/shared/meson.build
index bb7be41..e349ca4 100644
--- a/src/shared/meson.build
+++ b/src/shared/meson.build
@@ -625,4 +625,4 @@ libshared = shared_library(
 #endif // 0
         dependencies : libshared_deps,
         install : true,
-        install_dir : rootpkglibdir)
+        install_dir : rootlibexecdir)
